pkgs/test_core/lib/src/runner/configuration/args.dart

```diff
@@ -174,6 +174,22 @@ String get usage => _parser.usage;
 /// Throws a [FormatException] if [args] are invalid.
 Configuration parse(List<String> args) => _Parser(args).parse();
 
+PathConfiguration _parsePathConfiguration(String option) {
+  final uri = Uri.parse(option);
+
+  final name = uri.queryParameters['name'];
```

Comment:
I think for this one specifically we did want to allow multiple - you can support that using `uri.queryParametersAll['name']`